README.md

```diff
@@ -1,10 +1,26 @@
-# listenbrainz-server
-
-Server for the ListenBrainz project
-
-[Website](https://listenbrainz.org) |
-[Documentation](https://listenbrainz.readthedocs.io) |
-[Bug tracker](https://tickets.metabrainz.org/projects/LB/issues)
+<h1 align="center">
+  <br>
+  <a href="https://github.com/metabrainz/listenbrainz-server/archive/master.zip"><img src="https://github.com/metabrainz/metabrainz-logos/blob/master/logos/ListenBrainz/SVG/ListenBrainz_logo.svg" alt="ListenBrainz"></a>
```

Comment:
The link to a zip archive of the repo doesn't seem appropriate here.

Maybe link to LB.org instead? (I know we have it below as well, but…)

Reply:
Done